api/types.go

```diff
@@ -0,0 +1,47 @@
+package api
+
+type Context struct {
+	State DebuggerState `json:"state"`
```

Comment:
I'd just like to point out that context is pecifically used for request scoping in Go

https://blog.golang.org/context

Potentially a more specific term could be used